source/main.cpp

```diff
@@ -94,10 +95,11 @@ using namespace Aws::Iot::DeviceClient::SensorPublish;
 
 const char *TAG = "Main.cpp";
 
-vector<Feature *> features;
+// vector<Feature *> features;
```

Comment:
Don't leave commented code like this since it will be confusing and more difficult to read. I won't repeat that but it applies in the code that follows. Since we are using version control, we can always examine the previous code without needing to leave it commented in the source file.